apps/shipments/management/commands/replace_shipments.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import sys
+
+from django.core.management.base import BaseCommand
+
+from django.conf import settings
+from shipchain_common.exceptions import RPCError
+
+from apps.eth.models import EthAction
+from apps.eth.rpc import EventRPCClient
+from apps.jobs.models import AsyncJob
+from apps.jobs.tasks import AsyncTask
+from apps.shipments.models import Shipment
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+class Command(BaseCommand):
+    help = 'Moves shipment(s) to etherscan'
```

Comment:
This is not accurate, this command is being used to replicate shipment related transactions to the sidechain.